### ISSUE TITLE
Resume Nightly Builds

### ISSUE THREAD

**Issue by jimmyliang-gov** [ploigos/ploigos-containers#122]
Per the workflow:
This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs.

Can we please resume workflows here. I plan to be submitting PRs in the coming weeks for this project. Thanks!

**Comment by jimmyliang-gov** [comment]
looks like it failed. i'll poke around....

but as a general question - does the publish process enable entitled builds from RH?

**Comment by dwinchell** [comment]
No, the publish process consists entirely of the github workflow. There are no entitled VMs or build containers invovled.
